### ISSUE TITLE
Obsolete config options - preview_pane

### ISSUE THREAD

**Issue by dbieling** [roundcube/roundcubemail#5609]
Even after using the new config.inc.php-File without preview_pane, message is still visible. 
Version: Roundcube 1.3_beta
I tried it even in a new browser, to eliminate cache-interference. 

![roundcube_1 3_beta_obsoleteconfiguration](https://cloud.githubusercontent.com/assets/19514133/22040611/b01d19f0-dd03-11e6-85a9-28470be26343.png)

**Comment by alecpl** [comment]
Maybe you didn't update defaults.inc.php?

**Comment by dbieling** [comment]
Thanks for your response. I followed painstakingly upgrade instructions, but to be sure, I did replace the file defaults.inc.php (as you suggested) again. 
But maybe, it was a good choice, to delete the installation folder. This morning, i tried again. Uploaded installation folder.(last time, I just overwrote it) Went through the installation check, and everything went smoothly. 
Third possibility, roundcube enjoyed the night, to rest.  One of these three steps helped. 
Thanks you very much!!
